src/release/crater.md

```diff
@@ -0,0 +1,86 @@
+```
+
+Once the runs complete, you want to triage them
+
+## Triaging
+
+These steps should generally be done for the normal rustc run, and then followed up by a triage of
+the rustdoc run. Ignore failures in rustdoc that look to be rooted in rustc (i.e., duplicate
+failures).
+
+There will usually be quite a few regressions -- there are a couple tools that can help reduce the
+amount of work that you need to do. It's mostly a matter of personal preference which is more
+helpful.
+
+ * https://github.com/Mark-Simulacrum/crater-generate-report/
+   * This groups regressions by 'root' by parsing the logs to look for the compilation failed
+     messages printed by Cargo
+ * https://github.com/Centril/crater-cat-errors
+   * This groups regressions by the "error" message, also by parsing logs
+
+If you've written a tool, feel free to add it here! We're still figuring out what the best UI for
+this is.
+
+Regardless of the tool you've run, you ultimately need to read through a bunch of logs and try to
+quickly determine if they're genuine failures or spurious. Most of the time, a compiler failure is
+genuine, and test failures are mostly spurious, but this usually requires some level of guessing.
+
+Once you've determined that something is a genuine failure, add it to a list somewhere (local file,
+HackMD, whatever) with the error "category." Mostly, you're trying to group things such that the
+regressions in a single group are all caused by the same set of commits, and different groups have
+different causes.
+
+Once this is done, and you have all the regressions triaged into their separate groups, you want to
+file a new issue for each group. It should have the `regression-from-stable-to-beta` and
+`T-compiler` label by default, possibly `T-libs` if it's a standard library regression, but that's
+relatively rare. If you happen to think you know the PR that caused the failure, cc the PR author in
+a separate comment and link to the PR; otherwise compiler team will triage the issue soon.
+
+Leave a comment on the original issue with the crater runs linking to all the issues you just
+opened, ideally with the issue titles as well.
+
+You're done!
+
+## Re-running rustc on a crate
+
+For the crates which we're not sure about, you can try running crater locally, or build the crate
+directly ([`cargo-curl`](https://gist.github.com/nikomatsakis/75e09ed314bea715e192ce226127d3c6 ) can
+be helpful). Be careful -- regardless of what you do, you are running arbitrary code locally. It's
+also fine to file issues for the crates you're not sure about and let the triage process naturally
+categorize the error, though it's not good to do this for *all* the crates. Once you've triaged a
+crater run a couple times you get a pretty good sense of what is spurious and what isn't, too.
+
+You can run crater on just a single crate by doing something like this (at least, as of now).
```

Comment:
You should add a note that this requires Docker to be installed and running, and that it will download a few GBs of data on first use.